### DISCUSSION TITLE
bun:sqlite support multiple statements

### DISCUSSION THREAD

**Discussion by dominictobias** [oven-sh/bun#12819] [Q&A] [answered]
It seems that `bun:sqlite` can only run one statement at a time, but what about running a migration file for example?

**Comment by dominictobias** [comment, marked as answer]
The answer is to use `db.exec()` not `db.query().run` 😳

**Reply by DiveInto** [reply]
thanks, it is weird that this `.exec()` API is not mentioned at the api doc https://bun.sh/docs/api/sqlite
